README.md

```diff
@@ -79,18 +83,20 @@ Example Playbook
   roles:
     - name: storage
       storage_pools:
-        - name: "{{ app_name }}"
-          disks: "{{ app_data_wwns }}"
+        - name: app
+          disks:
+            - /dev/sdb
+            - /dev/sdc
```

Comment:
I would personally emphasize the convenience we provide by omitting the '/dev/' prefix on these.